### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by mikaEz**

### CODE REVIEW

File: .github/settings.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "repos": {
+    "description": "A Terraform module that manages the network resources from the azurerm provider.",
+    "homepage": "https://telekom-mms.github.io",
```
Comment (rndmh3ro) [inline .github/settings.json]: This does not exist and I'm not sure if anybody will ever create it. Is this used anywhere except on Github? If not, let's just omit it.

File: .github/workflows/linting.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: linting
+
+on: [push, pull_request]
```
Comment (rndmh3ro) [inline .github/workflows/linting.yml]: Let's do it like this:
```
on:
  push:
    branches: [master]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: [master]
```

This way, the job runs, when:
* there's a push to master OR
* there's a push to a pull-request (not just a branch) against master. This also avoids that the job runs twice on a pull-request.

File: .github/workflows/scan.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+name: scan
+
+on: [pull_request]
+
+env:
+  TEMPLATE_REPOSITORY: telekom-mms/terraform-template
+
+jobs:
+  call-scan:
```
Comment (rndmh3ro) [inline .github/workflows/scan.yml]:
```suggestion
  call-terrascan:
```

**Review by rndmh3ro — CHANGES_REQUESTED** [review]

File: .github/settings.json (hunk 1 above)
Comment (mikaEz) [reply .github/settings.json]: solved by https://github.com/telekom-mms/terraform-azurerm-network/pull/23/commits/eddf7ca51702c34692f101f8178c9d4c0b7609fe

File: .github/workflows/linting.yml (hunk 2 above)
Comment (mikaEz) [reply .github/workflows/linting.yml]: solved by https://github.com/telekom-mms/terraform-azurerm-network/pull/23/commits/eddf7ca51702c34692f101f8178c9d4c0b7609fe

File: .github/workflows/scan.yml (hunk 3 above)
Comment (mikaEz) [reply .github/workflows/scan.yml]: solved by https://github.com/telekom-mms/terraform-azurerm-network/pull/23/commits/eddf7ca51702c34692f101f8178c9d4c0b7609fe

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**